newsroom/agenda/formatters/csv_formatter.py

```diff
@@ -0,0 +1,126 @@
+from newsroom.formatter import BaseFormatter
+import csv
+import io
+import arrow
+from werkzeug.utils import secure_filename
+from newsroom.utils import parse_dates
+from datetime import datetime
+
+
+class CSVFormatter(BaseFormatter):
+    VERSION = "1.0"
+    PRODID = "Newshub"
+    FILE_EXTENSION = "csv"
+    MIMETYPE = "text/csv"
+    MULTI = True
+
+    def format_item(self, item, item_type=None):
+        event_item = self.format_event(item)
+        return self.serialize_to_csv(event_item)
+
+    def format_items(self, items, item_type=None):
+        formatted_event = []
+        for item in items:
+            parse_dates(item)
+            formatted_event.append(self.format_event(item))
+        return self.serialize_to_csv(formatted_event), secure_filename(
+            f"{datetime.now().strftime('%Y-%m-%d-%H:%M:%S')}-{'multi'}.{self.FILE_EXTENSION}"
+        )
+
+    def serialize_to_csv(self, data):
```

Comment:
would be good to use something more descriptive than `data`, it's not clear if it should be one item or a list